src/main/java/org/folio/consortia/repository/UserTenantRepository.java

```diff
@@ -14,6 +14,10 @@
 
 @Repository
 public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UUID> {
+
+  @Query("SELECT ut FROM UserTenantEntity ut WHERE ut.tenant.isDeleted IS NULL OR ut.tenant.isDeleted = FALSE")
+  Page<UserTenantEntity> getAll(Pageable pageable);
```

Comment:
existsByTenantId method now is not used any more, so can be deleted

Reply:
Thanks removed